R/main.R

```diff
@@ -29,17 +29,18 @@ collection <- function(collection_name, connection, column_names = character(),
                        post_process = I) {
   make_sure_table_exists(connection, collection_name)
 
-  query <- post_process(rethinker::r()$db(DEFAULT_DB)$table(collection_name))
+  tab_hand <- get_table_handle(connection$db_name, collection_name)
```

Comment:
I personally very much prefer full words in variable names instead of ambiguous abbreviations. How about writing simply `collection` or `table`? or `table_handle`? we don't loose much brevity but we gain a lot on redability.

btw I've noticed we're using table and collection to denote the same thing, I think we should clear this up and decide on one of them, but this is outside the scope of this PR